src/algebra/algebra/basic.lean

```diff
@@ -124,6 +124,74 @@ end prio
 def algebra_map (R : Type u) (A : Type v) [comm_semiring R] [semiring A] [algebra R A] : R →+* A :=
 algebra.to_ring_hom
 
+namespace algebra_map
+
+def has_coe_t (R A : Type*) [comm_semiring R] [semiring A] [algebra R A] :
+  has_coe_t R A := ⟨λ r, algebra_map R A r⟩
```

Comment:
does it matter if this is eta-expanded, or not?

Reply:
I assume it doesn't matter since eta-expansion of function is defeq anyway.